tests/python/pants_test/binaries/test_binary_util.py

```diff
@@ -350,7 +350,7 @@ def test_select_argv(self):
 [GLOBAL]
 allow_external_binary_tool_downloads: True
 pants_bootstrapdir: {}
-""".format(tmp_dir), binary_mode=False)
+""".format(tmp_dir))
```

Comment:
I think this will need a `mode` argument to avoid deprecation.

Reply:
Set to `mode='w'`!